src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsService.java

```diff
@@ -365,8 +346,45 @@ else if (hasFailures && testCase.getFailures().get(0).getType() != null) {
             result.addFeedbacks(scaFeedback);
         }
 
-        // Relevant feedback is negative
-        result.setHasFeedback(result.getFeedbacks().stream().anyMatch(fb -> !fb.isPositive()));
+        // Relevant feedback is negative, or positive with a message
+        result.setHasFeedback(result.getFeedbacks().stream().anyMatch(fb -> !fb.isPositive() || fb.getDetailText() != null));
+    }
+
+    /**
+     * Extracts the most helpful messages from the given test case.
```

Comment:
This method will always return one String in a list.
I suggest to return `String` then, otherwise it is confusing and other developers think that multiple strings can be returned